co2calculator/constants.py

```diff
@@ -124,3 +126,42 @@ class TransportationMode(str, enum.Enum):
     TRAM = "tram"
     BICYCLE = "bicycle"
     PEDELEC = "pedelec"
+
+
+class CountryCode2(str):
+    @classmethod
+    def __get_validators__(cls):
+        yield cls.validate_country_code
+
+    @classmethod
+    def validate_country_code(cls, country_code: str) -> str:
+        if country_code in list(iso3166.countries_by_alpha2.keys()):
+            return country_code
+        else:
+            raise ValueError(f"{country_code} is not a valid country code")
+
+
+class CountryCode3(str):
```

Comment:
Maybe add some docstirngs explaining the difference between CountryCode2 and CountryCode3?